src/PortfolioPlanning/Redux/Sagas/EpicTimelineSaga.ts

```diff
@@ -116,6 +116,22 @@ function* onAddEpics(action: ActionsOfType<EpicTimelineActions, EpicTimelineActi
         portfolioQueryInput
     );
 
+    let epicsWithoutDates:number[] = [];
+
+    queryResult.items.items.map(item => {
+      if(item.StartDate === undefined || item.TargetDate === undefined) {

```

Comment:
Do you have prettier on? Just load the workspace when you open code. Ed posted about it in the channel.

Reply:
I have the prettier on, forgot to run it before committing.